src/OrleansPSUtils/OrleansPSUtils.2017.csproj

```diff
@@ -0,0 +1,59 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup Label="NuGet">
+    <IsPackable>true</IsPackable>
+    <PackageId>Microsoft.Orleans.OrleansPSUtils</PackageId>
```

Comment:
I've other issues with the PS extension, since here we're using s 3rd party nuget package which contains a powershell dll...

With the new csproj's we don't have same set of variables set, I'll check it later.

This PR is the first step, not a complete replacement (mainly because of F#).